cli/go.sum

```diff
@@ -34,6 +34,7 @@ cloud.google.com/go/bigquery v1.8.0/go.mod h1:J5hqkt3O0uAFnINi6JXValWIb1v0goeZM7
 cloud.google.com/go/datastore v1.0.0/go.mod h1:LXYbyblFSglQ5pkeyhO+Qmw7ukd3C+pD7TKLgZqpHYE=
 cloud.google.com/go/datastore v1.1.0/go.mod h1:umbIZjpQpHh4hmRpGhH4tLFup+FVzqBi1b3c64qFpCk=
 cloud.google.com/go/firestore v1.1.0/go.mod h1:ulACoGHTpvq5r8rxGJ4ddJZBZqakUQqClKRT5SZwBmk=
+cloud.google.com/go/firestore v1.6.0/go.mod h1:afJwI0vaXwAG54kI7A//lP/lSPDkQORQuMkv56TxEPU=
```

Comment:
All the changes in this file are because of two reasons:
1. I added viper as a dependency (CI was failing) with
```sh
# In the keptn root directory
$ go get cli/pkg/config
```
2. I updated the dependency from viper 1.8.1 -> 1.9.0 in the `go.mod` because 1.9.0 is the latest stable version at the time of this comment.  1.8.1 was added automatically on running the `go get` command above. After updating the `go.mod` file I ran the same command again
```sh
# In the keptn root directory
$ go get cli/pkg/config
```